ui/components/compliance/compliance-card.tsx

```diff
@@ -105,6 +105,7 @@ export const ComplianceCard: React.FC<ComplianceCardProps> = ({
       shadow="sm"
       isPressable={isPressable}
       onPress={navigateToDetail}
+      as={"div"}
```

Comment:
We’ll probably find another way to fix this, but for now, to keep things moving, I’m happy with the changes to SelectScanProvider. Could you remove this change from the PR so we can go ahead and merge it? I’ll look into this part myself. What do you think?